t/t0036-broker-rundir.t

```diff
@@ -112,6 +112,20 @@ for attr in rundir statedir; do
 	test_must_fail flux start ${ARGS_NORC} flux setattr $attr x &&
 	test_must_fail flux start ${ARGS_NORC} flux setattr $attr-cleanup x
     '
+    test_expect_success "-S$attr=DIR interprets DIR sticky bit" '
+        dirname=`mktemp -d` &&
+	chmod 1777 $dirname &&
+	flux start ${ARGS_NORC} -S$attr=$dirname \
+		./checkattrs.sh >$attr-sticky.out
+    '
+    test_expect_success "$attr/DIR was used" '
+	grep "^$attr=$dirname/" $attr-sticky.out
+    '
+    test_expect_success "$attr/DIR was cleaned up but not $attr" '
+        subdirname=$(kv_lookup $attr $attr-sticky.out) &&
+	test $(dirname $subdirname) &&
+	test ! -e $subdirname
+    '
```

Comment:
Is it just the github diff or is there some missing tabs here?

Also, does `test $(dirname $subdirname)` do anything, do you need `test -d` instead? I'm actually not sure what happens with just `test /path/to/dir`, for me it just silently succeeds.

Reply:
Oh good catch on both!